### PR TITLE
bdd -> aiger.expr

### PR DESCRIPTION
**Pull request by mvcisback**
Talking to Eric, this seems like a desirable feature.

Implementation currently relies on parsing the output of dd's `to_expr`. While I'm not happy about this, a less jank solution didn't seem obvious using dd's interface.

### CODE REVIEW

File: aiger_analysis/test_count.py [hunk 1]
```diff
@@ -9,6 +9,22 @@
 from aiger_analysis import count
 
 
+@given(aigh.Circuits, st.data())
+def test_bdd_transform(circ, data):
+    circ = circ.unroll(3)
```
Comment (MarkusRabe) [inline aiger_analysis/test_count.py]: Why unroll? -> Separate unit test?

Comment (mvcisback) [reply aiger_analysis/test_count.py]: In general circ will have a latch. I just unrolled to remove the latch. I could have also used `cutlatches`.

**Comment by mvcisback** [comment]
@MarkusRabe I'll add an intermediate test that that tests evaluating the bdd. 
It's a bit tricky because dd doesn't have a simple evaluate command. AFAIK you either have to walk the decision diagram or conjunct with a cube.

File: aiger_analysis/test_count.py (hunk 1 above)
Comment (MarkusRabe) [reply aiger_analysis/test_count.py]: Ah, seems more natural, to use cutlatches, but I don't mind.

**Comment by MarkusRabe** [comment]
Can you test the equivalence of two BDDs? Maybe that would be another way.

**Comment by mvcisback** [comment]
Ok. Added test that bdd evaluation is correct in addition to the bijection. Merging now.

**Pull request merged**